src/SFA.DAS.ApplyService.Application/Apply/Moderator/SubmitModeratorPageOutcomeHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MediatR;
+using Microsoft.Extensions.Logging;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Moderator
+{
+    public class SubmitModeratorPageOutcomeHandler : IRequestHandler<SubmitModeratorPageOutcomeRequest>
+    {
+        private readonly IAssessorRepository _repository;
+        private readonly ILogger<SubmitModeratorPageOutcomeHandler> _logger;
+
+        public SubmitModeratorPageOutcomeHandler(IAssessorRepository repository, ILogger<SubmitModeratorPageOutcomeHandler> logger)
+        {
+            _repository = repository;
+            _logger = logger;
+        }
+
+        public async Task<Unit> Handle(SubmitModeratorPageOutcomeRequest request, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation($"SubmitModeratorPageOutcome for ApplicationId '{request.ApplicationId}' - " +
+                                                    $"SequenceNumber '{request.SequenceNumber}' - SectionNumber '{request.SectionNumber}' - PageId '{request.PageId}' - " +
+                                                    $"UserId '{request.UserId}' - Status '{request.Status}' - Comment '{request.Comment}' - ExternalComment '{request.ExternalComment}'");
```

Comment:
you might want to remove the comments from the log as they might be sensitive

Reply:
tweaked logging message